formatter/friendly.go

```diff
@@ -125,17 +124,19 @@ func (f *Friendly) printStatistics(w io.Writer, header string, stats map[string]
 		formatted = append(formatted, []string{color.GreenString(fmt.Sprintf("%d", entry.failures)), entry.name})
 	}
 	fmt.Fprintln(w, header)
-	fmt.Fprintln(w, f.table(formatted))
+	fmt.Fprintln(w, table(formatted))
 }
 
-func (*Friendly) table(rows [][]string) string {
-	buf := new(bytes.Buffer)
-	table := tablewriter.NewWriter(buf)
-	table.SetBorder(false)
-	table.SetColumnSeparator("")
-	table.SetRowSeparator("")
-	table.SetAutoWrapText(false)
-	table.AppendBulk(rows)
-	table.Render()
+func table(rows [][]string) string {
+	var buf strings.Builder
+	tw := tabwriter.NewWriter(&buf, 0, 0, 2, ' ', 0)
+	for _, row := range rows {
+		fmt.Fprintf(tw, "\t")
+		for _, col := range row {
+			fmt.Fprintf(tw, "%s\t", col)
+		}
+		tw.Write([]byte("\n"))
+	}
```

Comment:
It's a matter of style, but we can use the same `bytes.Buffer` and don't use `fmt.Fprintf`:
```suggestion
	buf := new(bytes.Buffer)
	tw := tabwriter.NewWriter(&buf, 0, 0, 2, ' ', 0)
	for _, row := range rows {
		tw.Write([]byte{'\t'})
		for _, col := range row {
			tw.Write(append([]byte(col), '\t'))
		}
		tw.Write([]byte{'\n'})
	}
```

Reply:
@alexandear idiomatically [bytes.Buffer needs no initialization](https://pkg.go.dev/bytes#Buffer), so it should be `var buf bytes.Buffer`.

Reply:
You're right. I left _buf := new(bytes.Buffer)_ because it was in the original version 🙂